meta-refkit/conf/layer.conf

```diff
@@ -67,3 +67,11 @@ LAYERDEPENDS_refkit = " \
     refkit-core \
     flatpak-layer \
 "
+
+# HAVE_NODEJS set here is a workaround needed to deal with signature
+# check failures with meta-refkit-core. mraa/upm recipes moved to
+# meta-oe accidentally kept HAVE_NODEJS check to enable nodejs bindings
+# but that changes the recipe defaults when HAVE_NODEJS is set in meta-refkit-core.
+# Therefore, only set HAVE_NODEJS in the distro configuration for now.
+HAVE_META_IOT_WEB = "${@ bb.utils.contains('BBFILE_COLLECTIONS', 'iot-web', 'True', 'False', d) }"
+HAVE_NODEJS ??= "${HAVE_META_IOT_WEB}"
```

Comment:
As a workaround that's okay. Will the recipes be changed to not use HAVE_NODEJS?

Reply:
Thanks. Yes, I'll submit a fix.